sake/redis.py

```diff
@@ -691,6 +740,17 @@ def intents(cls) -> hikari.Intents:
         # <<Inherited docstring from ResourceClient>>
         return hikari.Intents.NONE
 
+    @_as_tanjun_loader
+    def __add_to_tanjun(
+        self, client: tanjun.InjectorClient, trust_get_for: typing.Collection[typing.Type[ResourceClient]], /
+    ) -> None:
+        from tanjun.dependencies import async_cache
+
+        from . import _tanjun_adapter
+
+        adapter = _tanjun_adapter.SingleStoreAdapter(self.get_me, trust_get=UserCache in trust_get_for)
```

Comment:
TODO: how to handle trust_get for a hidden resource impl, maybe also allow the relevant abcs?